fisher.fish

```diff
@@ -43,6 +44,13 @@ function fisher -a cmd -d "fish package manager"
         command mv -f "$fisher_path/fishfile" "$fishfile"
     end
 
+    # 2020-06-23: temp code, migrates fisher data from XDG_CONFIG_HOME to XDG_DATA_HOME
+    set -l fisher_config $XDG_CONFIG_HOME/fisher
+    if test -d $fisher_config
+        command rm -rf $fisher_data
```

Comment:
How about a courtesy notice here to the user to let them be aware this happens?

Reply:
Nice! I'll get that in right away. 😊 👍

Reply:
@thernstig & @jorgebucaran, I added the notice. I was thinking about adding a note that it will / should happen only once, but after running another command, the user should be aware that it was only a one-time call. Let me know what you think. 🙂

Reply:
@jorgebucaran decides if he likes it or not :)

Reply:
What do you think, @jorgebucaran? Is the message good now, or do you want a note saying the migration will only happen once?

Reply:
Thank you @thernstig & @james-r-smith. The message is fine, thank you.